doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -419,19 +425,21 @@ Admin REST API provides the following actions:
 * [Get server-side endpoint profile schemas]({{root_url}}Programming-guide/Server-REST-APIs/#!/Profiling/getServerProfileSchemasByApplicationToken)
 * [Update server-side endpoint profile]({{root_url}}Programming-guide/Server-REST-APIs/#!/Profiling/updateServerProfile)
 
+For detailed description of the REST API, its purpose, interface, and features, see [server REST API]({{root_url}}Programming-guide/Server-REST-APIs/#/Profiling) its purpose, interfaces and features supported.
 
-## System part of endpoint profile ##
+## System part of EP profile
 
-The system part of the endpoint profile is used by Kaa internally for its functions and contains the following information.
+The system part of the endpoint profile is used by Kaa platform to perform internal functions.
+Below is the list of system part properties.
 
-| Property                |Description                                                                        | 
+| Property                |Description                                                                        |
 |-------------------------------------------------------------------------------------------------------------|
-|Server-side Profile Hash | The server-side profile hash used to validate integrity of endpoint profile state |
-|Server Key Hash          | The hash of the server that was used during the last endpoint request             |
-|SDK token                | The endpoint SDK identifier                                                       |
-|Endpoint Public Key      | The public key used for security purposes to validate endpoint requests           |
-|Endpoint group state     | The list of endpoint groups that contain current endpoint.                        |
-|Client-side Profile Hash | The client-side profile hash used to validate integrity of endpoint profile state |
+|Server-side profile hash | Server-side profile hash used to validate integrity of endpoint profile state.    |
+|Server key hash          | Server hash used during the last endpoint request.                                |
+|SDK token                | Endpoint SDK identifier.                                                          |
+|Endpoint public key      | Public key used for security purposes to validate endpoint requests.              |
+|Endpoint group state     | List of endpoint groups that contain current endpoint.                            |
+|Client-side profile hash | Client-side profile hash used to validate integrity of endpoint profile state.    |
 
 ## Further reading
 
```

Comment:
Well, they are connected but not all of them are _further_ reading, they are just related topics. I'm not sure we should have this section at the bottom of every page. I would delete this one.